From fb8feaf6ad8a8fb01c48e14c1069ce906803c697 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Mon, 16 Dec 2019 01:21:53 +0200
Subject: [PATCH 05/18] Pixel colors

  * from
  https://github.com/Havoc-OS/android_frameworks_base/commit/9f98b8ef947fcc6f7eea6690fd524807495367fb#diff-5cc79dde5c8fa0b4f33df52dc0799c70

	modified:   core/res/res/values/colors.xml
	modified:   core/res/res/values/colors_device_defaults.xml
	modified:   core/res/res/values/colors_material.xml
	modified:   core/res/res/values/dimens_material.xml
---
 core/res/res/values/colors.xml                 | 18 ++++++++--------
 core/res/res/values/colors_device_defaults.xml | 30 +++++++++++++-------------
 core/res/res/values/colors_material.xml        | 12 +++++------
 core/res/res/values/dimens_material.xml        |  2 +-
 4 files changed, 31 insertions(+), 31 deletions(-)

diff --git a/core/res/res/values/colors.xml b/core/res/res/values/colors.xml
index 1dcd389d9d8..75a428391bd 100644
--- a/core/res/res/values/colors.xml
+++ b/core/res/res/values/colors.xml
@@ -164,17 +164,17 @@
     <color name="accessibility_focus_highlight">#bf39b500</color>
     <color name="autofilled_highlight">#4dffeb3b</color>
 
-    <color name="system_notification_accent_color">#ff607D8B</color>
+    <color name="system_notification_accent_color">@android:color/accent_device_default</color>
 
     <!-- Default user icon colors -->
-    <color name="user_icon_1">#ff00bcd4</color><!-- cyan 500 -->
-    <color name="user_icon_2">#ff3f51b5</color><!-- indigo 500 -->
-    <color name="user_icon_3">#ff4285f4</color><!-- blue 500 -->
-    <color name="user_icon_4">#ffe91e63</color><!-- pink 500 -->
-    <color name="user_icon_5">#ff0f9d58</color><!-- green 500 -->
-    <color name="user_icon_6">#ff8bc34a</color><!-- light green 500 -->
-    <color name="user_icon_7">#ffff9800</color><!-- orange 500 -->
-    <color name="user_icon_8">#ffff5722</color><!-- deep orange 500 -->
+    <color name="user_icon_1">@android:color/accent_device_default</color>
+    <color name="user_icon_2">#ff5c6bc0</color>
+    <color name="user_icon_3">#ff26a69a</color>
+    <color name="user_icon_4">#ffec407a</color>
+    <color name="user_icon_5">#ff33ac71</color>
+    <color name="user_icon_6">#ff8bc34a</color>
+    <color name="user_icon_7">#ffff9800</color>
+    <color name="user_icon_8">#ffff7043</color>
     <color name="user_icon_default_gray">#ff9e9e9e</color><!-- gray 500 -->
     <color name="user_icon_default_white">#ffffffff</color><!-- white -->
 
diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 7a8f411992c..0c281b286d7 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -17,34 +17,34 @@
 <!-- Colors specific to DeviceDefault themes. These are mostly pass-throughs to enable
      overlaying new theme colors. -->
 <resources>
-    <color name="primary_device_default_dark">@color/primary_material_dark</color>
+    <color name="primary_device_default_dark">#ff2d2d2d</color>
     <color name="primary_device_default_light">@color/primary_material_light</color>
-    <color name="primary_device_default_settings">@color/primary_material_settings</color>
-    <color name="primary_device_default_settings_light">@color/primary_material_settings_light</color>
-    <color name="primary_dark_device_default_dark">@color/primary_dark_material_dark</color>
+    <color name="primary_device_default_settings">#ff2d2d2d</color>
+    <color name="primary_device_default_settings_light">#ffffffff</color>
+    <color name="primary_dark_device_default_dark">#ff242424</color>
     <color name="primary_dark_device_default_light">@color/primary_dark_material_light</color>
-    <color name="primary_dark_device_default_settings">@color/primary_dark_material_settings</color>
-    <color name="primary_dark_device_default_settings_light">@color/primary_dark_material_settings_light</color>
+    <color name="primary_dark_device_default_settings">#ff000000</color>
+    <color name="primary_dark_device_default_settings_light">#ffffffff</color>
 
     <color name="navigation_bar_divider_device_default_settings">#1f000000</color>
 
-    <color name="secondary_device_default_settings">@color/secondary_material_settings</color>
+    <color name="secondary_device_default_settings">#ff3a3a3a</color>
     <color name="secondary_device_default_settings_light">@color/secondary_material_settings_light</color>
-    <color name="tertiary_device_default_settings">@color/tertiary_material_settings</color>
-    <color name="quaternary_device_default_settings">@color/quaternary_material_settings</color>
+    <color name="tertiary_device_default_settings">#ff616161</color>
+    <color name="quaternary_device_default_settings">#ff9e9e9e</color>
 
-    <color name="accent_device_default_light">@color/accent_material_light</color>
-    <color name="accent_device_default_dark">@color/accent_material_dark</color>
+    <color name="accent_device_default_light">#ff1a73e8</color>
+    <color name="accent_device_default_dark">#ff8ab4f8</color>
     <color name="accent_device_default">@color/accent_device_default_light</color>
 
     <color name="background_device_default_dark">@color/background_material_dark</color>
-    <color name="background_device_default_light">@color/background_material_light</color>
-    <color name="background_floating_device_default_dark">@color/background_floating_material_dark</color>
+    <color name="background_device_default_light">#ffffffff</color>
+    <color name="background_floating_device_default_dark">#ff3c4043</color>
     <color name="background_floating_device_default_light">@color/background_floating_material_light</color>
 
     <!-- Error color -->
-    <color name="error_color_device_default_dark">@color/error_color_material_dark</color>
-    <color name="error_color_device_default_light">@color/error_color_material_light</color>
+    <color name="error_color_device_default_dark">#fff28b82</color>
+    <color name="error_color_device_default_light">#ffd93025</color>
 
     <color name="list_divider_color_light">#ffdadce0</color>
     <color name="list_divider_color_dark">#85ffffff</color>
diff --git a/core/res/res/values/colors_material.xml b/core/res/res/values/colors_material.xml
index d357f010277..7829e3b0636 100644
--- a/core/res/res/values/colors_material.xml
+++ b/core/res/res/values/colors_material.xml
@@ -39,8 +39,8 @@
     <color name="tertiary_material_settings">@color/material_blue_grey_700</color>
     <color name="quaternary_material_settings">@color/material_blue_grey_200</color>
 
-    <color name="accent_material_light">@color/material_deep_teal_500</color>
-    <color name="accent_material_dark">@color/material_deep_teal_200</color>
+    <color name="accent_material_light">@android:color/accent_device_default_light</color>
+    <color name="accent_material_dark">@android:color/accent_device_default_dark</color>
 
     <color name="button_material_dark">#ff5a595b</color>
     <color name="button_material_light">#ffd6d7d7</color>
@@ -113,9 +113,9 @@
 
     <color name="timepicker_default_background_material">@color/primary_text_default_material_light</color>
     <color name="timepicker_default_text_color_material">@color/black</color>
-    <color name="timepicker_default_ampm_selected_background_color_material">@color/material_deep_teal_200</color>
+    <color name="timepicker_default_ampm_selected_background_color_material">@android:color/accent_device_default_dark</color>
     <color name="timepicker_default_ampm_unselected_background_color_material">@color/transparent</color>
-    <color name="timepicker_default_selector_color_material">@color/material_deep_teal_200</color>
+    <color name="timepicker_default_selector_color_material">@android:color/accent_device_default_dark</color>
     <color name="timepicker_default_numbers_background_color_material">@color/transparent</color>
 
     <!-- DatePicker colors -->
@@ -139,8 +139,8 @@
     <color name="datepicker_default_pressed_text_color_material_light">#ff0099cc</color>
     <color name="datepicker_default_pressed_text_color_material_dark">#ff0099cc</color>
 
-    <color name="datepicker_default_circle_background_color_material_light">@color/material_deep_teal_500</color>
-    <color name="datepicker_default_circle_background_color_material_dark">@color/material_deep_teal_200</color>
+    <color name="datepicker_default_circle_background_color_material_light">@android:color/accent_device_default_light</color>
+    <color name="datepicker_default_circle_background_color_material_dark">@android:color/accent_device_default_dark</color>
 
     <color name="datepicker_default_view_animator_color_material_light">#fff2f2f2</color>
     <color name="datepicker_default_view_animator_color_material_dark">#ff303030</color>
diff --git a/core/res/res/values/dimens_material.xml b/core/res/res/values/dimens_material.xml
index 972fe7ed91d..500827f2af4 100644
--- a/core/res/res/values/dimens_material.xml
+++ b/core/res/res/values/dimens_material.xml
@@ -138,7 +138,7 @@
     <dimen name="seekbar_track_progress_height_material">2dp</dimen>
 
     <dimen name="progress_bar_height_material">4dp</dimen>
-    <dimen name="progress_bar_corner_material">0dp</dimen>
+    <dimen name="progress_bar_corner_material">1000dp</dimen>
 
     <!-- Material time picker dimensions. -->
     <!-- Text size for the time picker header HH:MM label. This value is large
-- 
2.11.0

